lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,47 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server:                 # https://webpack.js.org/configuration/dev-server/
+    host: localhost           # Dev server host
+    port: 8080
+    https: false
+
+  paths:                       # ~ = Rails.root
+    source: app/javascript     # ~/:source
+    entry: packs               # ~/:source/:entry
+    output: packs              # ~/public/:output
+    manifest: manifest.json    # ~/public/:output/:manifest
+    config: config/webpack     # ~/:config
```

Comment:
I'd like to propose a flattened set of options that drops `paths` entirely and helps clarifies each option's meaning (the fact that I added  the `~` legend is a smell):
```yaml
 source_path: app/javascript
 source_entry_path: packs
 public_output_path: packs
```
In doing so, we'd drop the `manifest` and `config` options. I'm not sure why you'd want/need to change those, but I'm open to keeping them if there's a compelling reason to.